battle_tanks/client/battle_tanks.cpp

```diff
@@ -0,0 +1,421 @@
+#include <iostream>
+#include <chrono>
+#include <memory>
+
+#include "SFML/System/Clock.hpp"
+#include "SFML/Graphics.hpp"
+#include <SFML/Network.hpp>
+#include <box2d/box2d.h>
+
+#include "utils/uuid.hpp"
+#include "game/game_objects/player_game_object.hpp"
+#include "game/sfml_game.hpp"
+#include "game/game_objects/bullet_game_object.hpp"
+#include "physics/collision_listener.hpp"
+#include "network/client_connection.hpp"
+#include "network/connection_service.hpp"
+#include "network/player_actions.hpp"
+#include "game/game_objects/forest_map_game_object.hpp"
+#include "game/game_objects/game_object_frame_restorer_packet.hpp"
+
+enum class player_game_state : unsigned int
+{
+    unknown = 0,
+    disconnected,
+    connecting,
+    main_lobby,
+    session_lobby,
+    game_session,
+};
+
+void set_text_color(sf::Text& text, const sf::Int32 current_hp)
+{
+    if (current_hp > 50)
```

Comment:
it's better to have all settings like this one in one or more dedicated config files for easier track of all constants and fast configuration of the game flow from one place. Possibly moving it all into some sort of db later to configure them outside of the code

Reply:
done